README.md

```diff
@@ -1,19 +1,78 @@
 # HDL Reference Designs
 
-[Analog Devices Inc.] HDL libraries and projects
+[Analog Devices Inc.](http://www.analog.com/en/index.html) HDL libraries and projects.
 
-### Branches
+## Getting started
 
-Each release has its own branch and master always synced with the latest release. 
-To find out more information about the latest release please check the release notes.
-Every branch, which has **dev** in its name, is a development branch and should handle it accordingly. 
+These projects currently supporting several different [FPGA development boards](https://github.com/analogdevicesinc/hdl/tree/master/projects) and [FMC boards](https://github.com/analogdevicesinc/hdl/tree/master/projects/common). 
+First make sure that the current state of the repository support's your hardware.
 
-### [Latest Release Notes]
-### [HDL User Guide]
-### [HDL Help & Support]
+### Prerequisites
 
-[HDL User Guide]:http://wiki.analog.com/resources/fpga/docs/hdl
-[HDL Help & Support]:http://ez.analog.com/community/fpga
-[Latest Release Notes]:https://github.com/analogdevicesinc/hdl/releases
-[Analog Devices Inc.]:http://www.analog.com/en/index.html
+ * [Vivado Design Suite](https://www.xilinx.com/support/download.html)
+
+**or**
+
+ * [Quartus Prime Design Suite](https://www.altera.com/downloads/download-center.html)
+ 
+Please make sure that you have the [required](https://github.com/analogdevicesinc/hdl/releases) tool version.
+
+### How to build a project
+
+For building a projects, you have to use the GNU Make tool. If you're a 
+Windows user please checkout this page, to see how you can install this tool.
```

Comment:
this page should have a link to hdl/build page ?

Reply:
It's there.